core/build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
     compile ('org.rnorth.duct-tape:duct-tape:1.0.7') {
         exclude(group: 'org.jetbrains', module: 'annotations')
     }
+    compile 'org.awaitility:awaitility:3.1.6'
```

Comment:
Cool, long time plan is to migrate away from duct-tape?

Reply:
Yep!

Reply:
but... as a public dependency, or shaded?

Reply:
Fair point - will shade it for safety.